maidr/patch/heatmap.py

```diff
@@ -13,12 +13,16 @@ def heat(wrapped, _, args, kwargs) -> Axes | AxesImage:
     # Check for additional label used by MAIDR heatmap.
     fill_label = kwargs.pop("fill_label", "Fill")
 
+    _fmt = ".2f"
```

Comment:
You can use the `fmt` variable, instead of `_fmt`

Reply:
@SaaiVenkat Renamed _fmt to fmt.